docs/requirements/ics10-requirements.md

```diff
@@ -1,98 +1,94 @@
 
 This is especially useful to protocols and companies whose business model is to improve cross-chain user interface, or to enable it when
 it's not. The main use case for this is bridging assets between chains. There are multiple protocols and companies currently performing such
 a service but none has yet been able to do it using IBC outside of the Cosmos ecosystem.
 
 A core piece for this to happen is to have a light client implementation of each ecosystem that has to be integrated, and uses a **new** consensus
-algorithm. This module broadens the horizon of light client development to not be limited to using Golang only for chains wanting use IBC and `ibc-go`,
-but instead expands the choice to any programming language and toolchain that is able to compile to wasm instead.
+algorithm. This module broadens the horizon of light client development to not be limited to using Golang only for chains wanting to use IBC and ibc-go,
+but instead expands the choice to any programming language and toolchain that is able to compile to Wasm instead.
 
-Bridging assets, is likely the simplest for of interchain communication. Its value is confirmed on a daily basis, when considering the volumes that protocols
+Bridging assets is likely the simplest form of interchain communication. Its value is confirmed on a daily basis, when considering the volumes that protocols
 like [Axelar](https://dappradar.com/multichain/defi/axelar-network), Gravity, [Wormhole](https://dappradar.com/multichain/defi/wormhole/) and
-[Layer0](TODO: add source for volume?) process. TODO: add sources for volume
-
+Layer0 process.
 
 ## Problem
 
-<!-- This section describes the problem that needs to be solved or the process that needs to be improved, as well as the environment in which the system will be used. This section could include a comparative evaluation of existing products, indicating why the proposed product is attractive and the advantages it provides. Describe the problems that cannot currently be solved without the envisioned solution. Show how it aligns with ecosystem trends, technology evolution, or strategic directions. List any other technologies, processes, or resources required to provide a complete solution. -->
-
-In order to export IBC outside of Tendermint based ecosystems, there is a need to introduce new light clients. This is a core need for
+In order to export IBC outside of Tendermint-based ecosystems, there is a need to introduce new light clients. This is a core need for
 companies and protocols trying to bridge ecosystems such as Ethereum, NEAR, Polkadot, etc. as none of these uses Tendermint as their
-consensus mechanism. Introducing a new light client implementation is not straightforwrd. The implementor needs to follow the light client's
+consensus mechanism. Introducing a new light client implementation is not straightforward. The implementor needs to follow the light client's
```

Comment:
perhaps we can add that the reason for this is that sometimes cryptographic primitives aren't available, or the same case for operating with certain data structures (like specific tries/trees, etc).